### PR TITLE
options: add -h (help), show complete options at usage()

### PR DESCRIPTION
**Pull request by avih**
I spent some time initially figuring out what the options were, so this makes them more discoverable.

### CODE REVIEW

**Comment by avih** [comment]
Force-pushed a typo fix `Unsage` -> `Usage`.

File: vt100/main.c [hunk 1]
```diff
@@ -110,7 +110,28 @@ static int cputhread (void *arg)
...
+void help(void)
+{
+  printf (
+    "Usage: %s [OPTIONS] command...\n"
+    "Run command [with arguments] in a hardware-emulated VT100.\n"
+    "\n"
+    "  -h      Display this help page and exit.\n"
+    "  -Q      Disable OpenGL and screen shader (may run faster).\n"
+    "  -N DIV  Reduce refresh-rate to 60/DIV Hz (may run faster).\n"
+    "  -2      Magnify the window. Each additional '-2' doubles the size.\n"
```
Comment (larsbrinkhoff) [inline vt100/main.c]: No exactly, it goes 2x, 3x, 4x, etc.

File: vt100/main.c [hunk 2]
```diff
@@ -110,7 +110,28 @@ static int cputhread (void *arg)
...
+    "  -f      Run in full-screen.\n"
+    "  -c CUR  Screen curvature (0.0 - 0.5, requires OpenGL).\n"
+    "  -C      Treat Caps-Lock as Control.\n"
+    "  -B      Treat backspace as rubout (currently not implemented).\n"
+    "  -R ROM  Use ROM file instead of the built-in VT100 rom.\n"
```
Comment (larsbrinkhoff) [inline vt100/main.c]: -R is more like "run CP/M executable".

**Comment by larsbrinkhoff** [comment]
This is great, thanks!  Just some minor comments.

**Comment by avih** [comment]
> No exactly, it goes 2x, 3x, 4x, etc.

Oops.. right. Fixed.

> -R is more like "run CP/M executable".

Interesting. I updated the text to something like this, but how is this related to CP/M? as far as I can tell you just take a binary file and execute it on the 8080 after some simple setup?

**Comment by larsbrinkhoff** [comment]
> how is this related to CP/M?

It's only for testing the 8080 CPU.  There are a few test suites under the test directory.

**Comment by larsbrinkhoff** [comment]
Your updates look good, thanks!  Merging.

**Comment by avih** [comment]
Thanks.

**Pull request merged**